### DISCUSSION TITLE
How to request a release bump?

### DISCUSSION THREAD

**Discussion by sunnycarter** [vmware/govmomi#3009] [Community Help]
Hi there,
I have a PR in a dependent repo that requires a bump and release of this repo (https://github.com/hashicorp/terraform-provider-vsphere/pull/1417) and has been waiting for some months for this.

Is there a process to request a release bump to unblock my PR?  I'm not familiar with what that process is I'm afraid.

This is the PR on this codebase that is merged that I'd like the bump to include:
https://github.com/vmware/govmomi/pull/2957 

Thanks

**Comment by sunnycarter** [comment]
Wondering if @dougm  can help as you reviewed and merged my PR??

**Reply by akutz** [reply]
Howdy @sunnycarter , I believe you should be good-to-go with release [v0.30.0](https://github.com/vmware/govmomi/releases/tag/v0.30.0) 😄

**Reply by tenthirtyam** [reply]
[`vmware/govmomi@v0.30.0`](https://github.com/vmware/govmomi/releases/tag/v0.30.0) has been release today and noted on the PR.

**Reply by tenthirtyam** [reply]
Well get a PR to update the Terraform provider soon so that we can finish testing https://github.com/hashicorp/terraform-provider-vsphere/pull/1417. :)
